commands/namespaces/add.go

```diff
@@ -39,11 +40,29 @@ func NewAddCommand(l *zap.SugaredLogger) *cobra.Command {
 			}
 			bindInstallHelmOpts(c)
 
-			if len(args) != 1 {
-				output.PrintError(fmt.Errorf("invalid number of arguments: expected 1, got %d", len(args)), l, true)
+			if len(args) == 1 {
+				// user provided namespace name
+				c.Namespaces = args[0]
+			} else if c.SkipWizard {
+				// user didn't provide namespace name, but we can't ask him to provide it
+				output.PrintError(namespaces.ErrNamespaceIsMissed, l, true)
 				os.Exit(1)
+			} else {
+				// ask user to provide namespace name
+				var resp string
+				q := &survey.Input{Message: "Enter namespace name"}
+				if err = survey.AskOne(q, &resp); err != nil {
+					output.PrintError(err, l, true)
+					os.Exit(1)
+				}
+
+				if resp == "" {
+					// user didn't provide namespace name
+					output.PrintError(namespaces.ErrNamespaceIsMissed, l, true)
+					os.Exit(1)
+				}
+				c.Namespaces = resp
```

Comment:
If you see below, there is already a function call `c.Populate` that can run this namespace name prompt if the second arg is set to `true` ... I think we should re-use that

Reply:
we decided to change help message and keep current logic as is.